NitroxPatcher/Patches/Dynamic/StasisSphere_OnHit_Patch.cs

```diff
@@ -0,0 +1,35 @@
+using System.Reflection;
+using NitroxClient.Communication.Abstract;
+using NitroxClient.GameLogic;
+using NitroxModel.DataStructures.Unity;
+using NitroxModel.Helper;
+using NitroxModel.Packets;
+using NitroxModel_Subnautica.DataStructures;
+
+namespace NitroxPatcher.Patches.Dynamic;
+
+/// <summary>
+/// Prevents remote stasis sphere from triggering hit effets (they're triggered with packets)
```

Comment:
I find the wording here a bit misleading as we don't skip the base-method din the prefix

Reply:
Fixed the patch to actually do what the summary explained.